docs/en/Getting-Started.md

```diff
@@ -1,81 +1,365 @@
+S3.
+
+## [AWS Bonus] Accelerating Data Access with Alluxio
+
+Since Alluxio leverages memory to store data, it can accelerate access to data. First, let’s take a
+look at the status of a file in Alluxio (mounted from S3).
+
+```bash
+$ ./bin/alluxio fs ls /mnt/s3/sample_tweets_150m.csv
+149.77MB  06-20-2016 12:50:55:473  Not In Memory  /mnt/s3/sample_tweets_150m.csv
+```
+
+The output shows that the file is **Not In Memory**. This file is a sample of tweets. Let’s see how
+many tweets mention the word "kitten". With the following command, we can count the number of
+tweets with "kitten".
+
+```bash
+$ time ./bin/alluxio fs cat /mnt/s3/sample_tweets_150m.csv | grep -c kitten
+889
+
+real	0m22.857s
+user	0m7.557s
+sys	0m1.181s
+```
+
+Depending on your network connection, the operation may take over 20 seconds. If reading this file
+takes too long, you may use a smaller dataset. The other files in the directory are smaller subsets
+of this file.
+
+Now, let’s see how many tweets mention the word "puppy".
+
+```bash
+$ time ./bin/alluxio fs cat /mnt/s3/sample_tweets_150m.csv | grep -c puppy
+1553
+
+real	0m25.998s
+user	0m6.828s
+sys	0m1.048s
+```
+
+As you can see, it takes a lot of time to access the data for each command. Alluxio can accelerate
+access to this data by using memory to store the data. However, the `cat` shell command prevents
+Alluxio from caching the data in memory. There is a separate shell command, `load`, which tells
+Alluxio to store the data in memory. You can tell Alluxio load the data into memory with the
+following command.
+
+```bash
+$ ./bin/alluxio fs load /mnt/s3/sample_tweets_150m.csv
+```
+
+After loading the file, you can check the status with the ls command:
+
+```bash
+$ ./bin/alluxio fs ls /mnt/s3/sample_tweets_150m.csv
+149.77MB  06-20-2016 12:50:55:473  In Memory      /mnt/s3/sample_tweets_150m.csv
+```
+
+The output shows that the file is now **In Memory**. Now that the file is memory, reading the file
+should be much faster now.
+
+Let’s count the number of tweets with the word "puppy".
+
+```bash
+$ time ./bin/alluxio fs cat /mnt/s3/sample_tweets_150m.csv | grep -c puppy
+1553
+
+real	0m1.917s
+user	0m2.306s
+sys	0m0.243s
+```
+
+As you can see, reading the file was very quick, only a few seconds! And, since the data in Alluxio
+memory, you can easily read the file again just as quickly. Let’s now count how many tweets mention
+the word "bunny".
+
+```bash
+$ time ./bin/alluxio fs cat /mnt/s3/sample_tweets_150m.csv | grep -c bunny
+907
+
+real	0m1.983s
+user	0m2.362s
+sys	0m0.240s
+```
+
+Congratulations! You installed Alluxio locally and used Alluxio to accelerate access to data!
+
+## Stopping Alluxio
+Now that you have installed and run Alluxio locally, and ran a few simple examples, you can stop
+Alluxio with the following command:
+
+```bash
+$ ./bin/alluxio-stop.sh all
+```
+
+## Next Steps
+
+Now that you have successfully started and interacted with Alluxio locally, there are several next
```

Comment:
This sounds weird since the previous section started with an almost identical sentence.